python/metatensor-learn/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 #### Fixed
 
+- Removal of usage of Labels.range in nn modules to support torch.jit.save see metatensor issue #410
+
```

Comment:
This should be added below, not inside the "possible sections" comment.

Reply:
So below the `-->` ?

Reply:
yes